layouts/_default/baseof.html

```diff
@@ -70,9 +70,19 @@
   {{ partial "pixels/gtm" .}}
   {{ end }}
 
-  <link rel="shortcut icon" href="{{site.BaseURL}}favicon-32x32.png" type="image/png">
+  <link rel="icon" href="/favicon.ico" sizes="any">
+  <link rel="icon" href="/favicon.svg" type="image/svg+xml">
+  <link rel="apple-touch-icon" href="/apple-touch-icon.png">
```

Comment:
Add base URL to all links: `<link rel="apple-touch-icon" href="{{site.BaseURL}}apple-touch-icon.png">`